core/templates/components/shared-component.module.ts

```diff
@@ -148,6 +151,7 @@ import { HybridRouterModuleProvider } from 'hybrid-router-module-provider';
     HybridRouterModuleProvider.provide(),
     MaterialModule,
     NgBootstrapModule,
+    NgbModule,
```

Comment:
why this module is imported?

Reply:
for using ngbPopover

Reply:
removed now

Reply:
I have used ngbpopover so imported NgbModule, but now it removed.